.github/workflows/code-review-templates/code-review-full.md

```diff
@@ -0,0 +1,57 @@
+## Code Review Checklist
+
+This is an automated comment on every pull request requiring a review. A checked item is either an assertion that I have tested this item or an indication that I have verified it does not apply to this pull request.
+
+## The Basics
+- [ ] Checks are passing
+- [ ] I read the code
+- [ ] I ran the code
+- [ ] (if applicable) Post deploy steps are run
+- [ ] (if applicable) I validated the change on the deployed version in <environment>
```

Comment:
I don't think I understand these. If we're reviewing it, doesn't that mean it hasn't been deployed yet? Or is this in the case of temporary deploys (a la cloud.gov Pages)?

Reply:
Yep! temporary